clog/clog.go

```diff
@@ -0,0 +1,196 @@
+/*
+Package clog provides Conext with logging information.
+*/
+package clog
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"sync"
+
+	"github.com/golang/glog"
+)
+
+// unique type to prevent assignment.
+type clogContextKeyT struct{}
+
+var clogContextKey = clogContextKeyT{}
+
+const (
+	// standard keys
```

Comment:
> What is the meaning of a standard key? 

These constants exists only for one reason - to be able to print them in sorted order.

> Why orchSessionID is not a standard key?

It is. I just forgot to add it to the `stdKeysOrder` array.

Reply:
They are 'standard' because we define them in the package, and have separate `AddXXX` methods for them - as opposed to general `AddVal` method which allows user to add any key.